packages/broker/test/integration/plugins/operator/OperatorPlugin.test.ts

```diff
@@ -23,7 +23,7 @@ describe('OperatorPlugin', () => {
             privateKey: brokerWallet.privateKey,
             extraPlugins: {
                 operator: {
-                    operatorContractAddress: '0x4A5C0EC07F7ddBd4B6050181638e24b0153991b2'
+                    operatorContractAddress: '0x139dfa493a45364b598f2f98e504192819082c85'
```

Comment:
Why this is changed?

Reply:
It was an outdated address (not anymore part of the config) after the fast-chain started to be used. Not sure how ci skipped it and got into streamr-1.0 (no impl for the operator plugin service was added in this PR). The new address is the one that's now always deployed into the fast-chain container.